From 973208cc6b5806c2f4091bd74503d624c406d869 Mon Sep 17 00:00:00 2001
From: Uwe Sauter <uwe.sauter.de@gmail.com>
Date: Sat, 27 Apr 2019 13:52:04 +0200
Subject: [PATCH 2/4] Functions qc_omx_component_tunnel_request,
 OMX_SetupTunnel and OMX_SetupTunnel exist as interface but do not implement
 any functionality. Mark function arguments with __attribute__((unused)) to
 get rid of compiler warnings (-Wunused-parameter).

Change-Id: I5e4a7bd028ce97cfb103308cfc325acf99db8c1a
---
 mm-core/src/common/omx_core_cmp.cpp | 10 +++++-----
 mm-core/src/common/qc_omx_core.c    | 12 ++++++------
 2 files changed, 11 insertions(+), 11 deletions(-)

diff --git a/mm-core/src/common/omx_core_cmp.cpp b/mm-core/src/common/omx_core_cmp.cpp
index a11cc4c2f..139335c1b 100755
--- a/mm-core/src/common/omx_core_cmp.cpp
+++ b/mm-core/src/common/omx_core_cmp.cpp
@@ -235,11 +235,11 @@ qc_omx_component_get_state(OMX_IN OMX_HANDLETYPE  hComp,
 }
 
  OMX_ERRORTYPE
-qc_omx_component_tunnel_request(OMX_IN OMX_HANDLETYPE                hComp,
-                       OMX_IN OMX_U32                        port,
-                       OMX_IN OMX_HANDLETYPE        peerComponent,
-                       OMX_IN OMX_U32                    peerPort,
-                       OMX_INOUT OMX_TUNNELSETUPTYPE* tunnelSetup)
+qc_omx_component_tunnel_request(__attribute__((unused)) OMX_IN OMX_HANDLETYPE hComp,
+                       __attribute__((unused)) OMX_IN OMX_U32                 port,
+                       __attribute__((unused)) OMX_IN OMX_HANDLETYPE          peerComponent,
+                       __attribute__((unused)) OMX_IN OMX_U32                 peerPort,
+                       __attribute__((unused)) OMX_INOUT OMX_TUNNELSETUPTYPE* tunnelSetup)
 {
   DEBUG_PRINT("Error: qc_omx_component_tunnel_request Not Implemented\n");
   return OMX_ErrorNotImplemented;
diff --git a/mm-core/src/common/qc_omx_core.c b/mm-core/src/common/qc_omx_core.c
index c8b560cf3..a8520fd54 100755
--- a/mm-core/src/common/qc_omx_core.c
+++ b/mm-core/src/common/qc_omx_core.c
@@ -552,10 +552,10 @@ RETURN VALUE
   None.
 ========================================================================== */
 OMX_API OMX_ERRORTYPE OMX_APIENTRY
-OMX_SetupTunnel(OMX_IN OMX_HANDLETYPE outputComponent,
-                OMX_IN OMX_U32             outputPort,
-                OMX_IN OMX_HANDLETYPE  inputComponent,
-                OMX_IN OMX_U32              inputPort)
+OMX_SetupTunnel(__attribute__((unused)) OMX_IN OMX_HANDLETYPE outputComponent,
+                __attribute__((unused)) OMX_IN OMX_U32        outputPort,
+                __attribute__((unused)) OMX_IN OMX_HANDLETYPE inputComponent,
+                __attribute__((unused)) OMX_IN OMX_U32        inputPort)
 {
   /* Not supported right now */
   DEBUG_PRINT("OMXCORE API: OMX_SetupTunnel Not implemented \n");
@@ -575,8 +575,8 @@ RETURN VALUE
   None.
 ========================================================================== */
 OMX_API OMX_ERRORTYPE
-OMX_GetContentPipe(OMX_OUT OMX_HANDLETYPE* pipe,
-                   OMX_IN OMX_STRING        uri)
+OMX_GetContentPipe(__attribute__((unused)) OMX_OUT OMX_HANDLETYPE* pipe,
+                   __attribute__((unused)) OMX_IN OMX_STRING       uri)
 {
   /* Not supported right now */
   DEBUG_PRINT("OMXCORE API: OMX_GetContentPipe Not implemented \n");
-- 
2.17.1

